code/game/objects/structures/beds_chairs/sofa.dm

```diff
@@ -9,7 +9,8 @@
 /obj/structure/chair/sofa/Initialize(mapload)
 	armrest = mutable_appearance(icon, "[icon_state]_armrest", ABOVE_MOB_LAYER)
 	armrest.plane = GAME_PLANE_UPPER
-	return ..()
+	. = ..()
```

Comment:
why would you do this

Reply:
I liked it more especially with initialize procs for before and after an object does its parent init code

Reply:
Me and Novaa are a part of "`return ..()` at the end of procs" gang

Reply:
i mostly just hate the fact that its placed in the middle of the actual code

Reply:
put it at the top 4head

Reply:
I will make it return ..() in situations where it is at the end of the proc but it isn't